relay/polkadot/src/genesis_config_presets.rs

```diff
@@ -135,6 +135,73 @@ fn default_parachains_host_configuration() -> HostConfiguration<polkadot_primiti
 	}
 }
 
+#[cfg(feature = "zombie-bite-sudo")] // FAIL-CI
+#[allow(clippy::type_complexity)]
+fn polkadot_testnet_genesis(
```

Comment:
We dont really use genesis anyway? Like we just overwrite it in ZB, or?

Reply:
That correct, we can remove this.